File: python/S1_BASIC/L4_BASICMATHS/divisors.py
class solution:
    def __init__(self,num):
        self.num=num
    def divisors(self):
        for i in range(1,self.num+1):
            if self.num%i==0:
                print(i,end=" ")
if __name__=="__main__":
    num=int(input("enter number: "))
    s=solution(num)
    s.divisors()

File: JAVA/S1_BASIC/L4_BASICMATHS/divisors.java
class solution{
    void divisors(int num){
        for(int i=1;i<=num/2;i++){
            if(num%i==0){
                System.out.print(i+" ");
            }
            
        }
        System.out.print(num);
    }
}
public class divisors{
    public static void main(String[]args){
        solution s=new solution();
        s.divisors(6034);
    }
}